ui/src/forms/BulmaRadioButtons.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <b-field :message="message">
+    <b-radio-button
+      v-for="option in choices"
+      :key="option.value"
+      :value="value.value"
+      :native-value="option.value"
+      @input="emit"
+    >
+      {{ label(option) }}
+    </b-radio-button>
+  </b-field>
+</template>
+
+<script lang="ts">
+import { Prop, Component, Vue } from 'vue-property-decorator'
+import { GraphPointer, MultiPointer } from 'clownface'
+import { Term } from 'rdf-js'
+import { rdfs } from '@tpluscode/rdf-ns-builders'
+
+@Component
+export default class extends Vue {
+  @Prop() options?: MultiPointer;
+  @Prop() value?: GraphPointer;
+  @Prop() update!: (newValue: Term | string) => void;
+
+  get choices () {
+    return this.options?.terms
+  }
+
+  message = ''
+
+  mounted () {
+    this.message = this.value?.out(rdfs.comment).value || ''
+  }
+
+  label (value: Term) {
+    return this.options?.node(value).out(rdfs.label).values[0] || value.value
+  }
+
+  emit (value: string) {
+    this.message = this.options?.namedNode(value).out(rdfs.comment).values[0] || ''
+    this.update(value)
+  }
```

Comment:
By trying `rdfs.label` and `rdfs.comment` we display hints where enum elements are named nodes. Strings are just strings

Reply:
Of course! I don't know why it wasn't obvious to me